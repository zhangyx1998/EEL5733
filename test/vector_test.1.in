+ 123321        #0
- 0
+ world         #0
+ hello         #1
+ brown         #2
+ fox           #3
- 2
- 2
+ fox           #2
+ brown         #3
+ a quick       #4
+ this is magic #5
