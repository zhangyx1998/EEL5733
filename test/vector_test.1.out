123321        #0
brown         #2
fox           #3
====>> EOF <<====
this is magic #5
a quick       #4
brown         #3
fox           #2
hello         #1
world         #0
